projects/core/src/lib/components/home-page/home-page.component.scss

```diff
@@ -0,0 +1,62 @@
+%title {
```

Comment:
YAGNI. 
do we need that variable if we only use it once in the code?

%separator the same

Reply:
Resolved